tools/cache.py

```diff
@@ -185,16 +188,27 @@ def get(shortname, creator, what=None, force=False, quiet=False, deferred=False)
   return str(cachename)
 
 
+def setup_file(cache_file):
+  global cachedir, cache_file_locks, acquired_count
```

Comment:
None of these looks like they get assigned in this function so I think you can delete this line.